### ISSUE TITLE
Unable to set export action to users

### ISSUE THREAD

**Issue by klementrempus** [nilsteampassnet/TeamPass#2299]
Hello,

I am unable to see the export option. I have enabled "Enable printing items to PDF file" with "YES". I only see the import optios, in the field "Define the roles that are allowed to print items" there are all the roles listed, but i haven't found a way to select or enable some of the roles, clicking, double clicking or pressing any keys does nothing, do the users from the list have to be selected additionally or are those all the users listed who should have the option to export? Please see screenshots.

![1](https://user-images.githubusercontent.com/39908729/41272642-3fe2f4de-6e16-11e8-887d-cef5edf91d9f.png)
![2](https://user-images.githubusercontent.com/39908729/41272643-3ffc43a8-6e16-11e8-8564-d104867b6f58.png)

Thank you,

BR, Klemen

**Comment by nilsteampassnet** [comment]
You need to select the roles that you allow to use this feature.
To select, just do as with all multi-select lists, use the mouse and CTRL keyboard key.

![image](https://user-images.githubusercontent.com/1197546/41496211-1d2d705a-713a-11e8-9f70-903454fecbc4.png)

**Comment by klementrempus** [comment]
Hello,

I have already tried this, I have created a role specifically for the purpose of exporting, named "Export" and granted it the RW rights to every folder and assigned it to a user and I am still unable to see the export action, please see screens below.

![image](https://user-images.githubusercontent.com/39908729/41521419-676321ee-72d2-11e8-9ae5-d7bff6a1a3ac.png)

![image](https://user-images.githubusercontent.com/39908729/41521424-6b909ada-72d2-11e8-804c-ea21deae268c.png)

![image](https://user-images.githubusercontent.com/39908729/41521475-982be996-72d2-11e8-9648-1398cd107459.png)

![image](https://user-images.githubusercontent.com/39908729/41521488-a271b08e-72d2-11e8-8dac-3fe0b673b990.png)

Is there something else that should be done? Somehow to confirm the selection?

BR, Klemen

**Comment by nilsteampassnet** [comment]
What version of Teampass?

**Comment by mborrasb** [comment]
i have the same issue, selecting the option " Export to PDF or CSV enabled", selecting role (or roles) the switch remain activated, but when i navigate and come back to the Options tab, it's switched off

Option "Import from KeePass or CSV files " seems work properly. I don't know if its missing to acitvate some other option to maintain this export active.

versión 2.1.27.16

![screenshot1](https://user-images.githubusercontent.com/42610539/44465080-0ba54900-a60c-11e8-94c7-9777b1eed192.jpg)
![screenshot2](https://user-images.githubusercontent.com/42610539/44465081-0ba54900-a60c-11e8-99c6-261daeeee97e.jpg)

**Comment by frachop** [comment]
Any news ?
I got same issue here (v 2.1.27.36 / ubuntu)

**Comment by necarnot** [comment]
Hello Nils,

Using TP 3.0.10, I am able to select the roles that will be allowed to get the option.
But, though I select the appropriate roles, they don't seem to be saved.
Quitting the options page to another page then returning to the options page is showing that the chosen roles were forgotten.

**Comment by necarnot** [comment]
Indeed, After adding some roles, I see no attempt to save these options (the usual bottom-right saving disappearing popup is not showing).
I also tried to disable then enable the export feature : that allows the saving popup to appear, but anyway, the roles are not saved neither.

**Comment by stvilla** [comment]
I'm having the same issue...

**Comment by necarnot** [comment]
Turns out it still allows users to get the export menu and use it successfully.
Still something that ought to be fixed, though.

**Comment by stvilla** [comment]
Hello @necarnot, I cannot find the export menù...

The screenshot shows the options of our Teampass:
https://github.com/nilsteampassnet/TeamPass/issues/2299#issuecomment-1711199648

If I log in as a user, I don't see the export option for folders:
 
![image](https://github.com/nilsteampassnet/TeamPass/assets/26825929/4952219e-6428-47e2-a893-b39e70e5dba5)

Do you have any suggestions?

**Comment by necarnot** [comment]
Buongiorno @stvilla 

Just like you, I was also expecting to find the export option appear at the location you outlined.
Actually, it appeared there :
![2023-09-08_11-10](https://github.com/nilsteampassnet/TeamPass/assets/4737235/987c401e-f501-4928-ad47-ce329ff091fa)

**Comment by stvilla** [comment]
Hello @necarnot,

I cannot see the option in the sidebar :sob: :sob: 
![image](https://github.com/nilsteampassnet/TeamPass/assets/26825929/4a337716-dc4b-407d-ac36-30e7e8964538)

I double-checked that the "enable export" flag Is on, but still, I cannot see the option...

**Comment by eferlin** [comment]
Hi @stvilla , 
Even enabling the option to export to CSV, and marking the rules I want to export, I don't see either IMPORT or EXPORT. I've already changed it to user, manager, viewer only and nothing.
I'm already going crazy, I've been looking for how to do this for so long and still nothing.

**Comment by eferlin** [comment]
Even enabling the option to export to CSV, and marking the rules I want to export, I don't see either EXPORT. I've already changed it to user, teampass manager, administrator, viewer only and nothing.
I'm already going crazy, I've been looking for how to do this for so long and still nothing.

However, I found something that you can test (it didn't work here).
Click on the import option and change the URL to export.
It didn't export anything, but it's already an advance. Can anyone test?

![image](https://github.com/nilsteampassnet/TeamPass/assets/39979300/09189402-129b-4d42-8c99-a29f8a9391e2)

**Comment by juliajulia-89** [comment]
hello everyone, 
I dont have problems exporting folders but I cant seem to find a way to export users
is there anyone who can help?
thank you
